inference-engine/tests/ngraph_functions/include/ngraph_functions/subgraph_builders.hpp

```diff
@@ -212,6 +209,16 @@ static std::shared_ptr<ngraph::Function> make2InputSubtract(std::vector<size_t>
     auto result = std::make_shared<ngraph::opset1::Result>(subtract);
     return std::make_shared<ngraph::Function>(ngraph::ResultVector{result}, ngraph::ParameterVector{param0, param1});
 }
+
+static std::shared_ptr<ngraph::Function> make2InputSubtractIn(std::vector<size_t> inputShape, InferenceEngine::Precision prc) {
```

Comment:
Looks like duplication of previous function

Reply:
No, this is a function with dynamic parameters.